Simple Inventory Management System/Simple Inventory Management System/Inventory .cs

```diff
@@ -0,0 +1,126 @@
+﻿
+namespace Simple_Inventory_Management_System
```

Comment:
Solution name, project name and namespaces should be in PascalCase, no spaces. Lets update all the names including `.sln` and `.csproj`.